src/ui/osx/TogglDesktop/AboutWindowController.m

```diff
@@ -12,6 +12,11 @@
 #import "DisplayCommand.h"
 #import "Utils.h"
 #import "Sparkle.h"
+#import "UnsupportedNotice.h"
+
+@interface AboutWindowController ()
+@property BOOL unsupportedOS;
```

Comment:
Ideally, `AboutWindowController` shouldn't have a knowledge about the OS version or unsupportedOS logic. I suggest that `UnsupportedNotice` should be in charge.

As a result, this logic should be moved to `UnsupportedNotice` as well.
https://github.com/toggl/toggldesktop/blob/c52da165d527d423f208cb63a65c1826923773fb/src/ui/osx/TogglDesktop/AboutWindowController.m#L48

Reply:
On the other hand, it would be nice if we start acquiring new modern objc-c style.
It means that it should be
```obj-c
// UnsupportedNotice.m
@property (nonatomic, assign) BOOL unsupportedOS;

// UnsupportedNotice.h
@property (nonatomic, assign, readonly) BOOL unsupportedOS;
```
Make sure that the public interface of `unsupportedOS` is only `readable`. We don't want other guys changed it by accident.